src/glossary.rst

```diff
@@ -3927,7 +3927,8 @@ module
 ^^^^^^
 
 :dp:`fls_ujlsg58bskl5`
-A :dt:`module` is a container for zero or more :t:`[item]s`.
+A :dt:`module` is a container for zero or more :t:`[inner attribute]s`,
+:t:`[inner doc comment]s`, and :t:`[item]s`.
```

Comment:
I don't quite like this, a module doesn't really contain attributes and doc comments in the sense that they contain items. The attributes and doc comments are part of the module, whether they are outer or inner ones.

Reply:
Fair enough. I restored the original wording.